City-of-Helsinki/drupal-hdbt
# UHF-6848: School feature search

# [UHF-6848](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6848)
Enables searching schools by name or postal code.

## What was done
* Added new tab to school search paragraph
* Added logic for searching schools by name or postal code
* Added some new shared components
* Added styles

## How to install / test

* Check info here: https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/322


[UHF-6848]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-6848?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
